app/controllers/api_guard/tokens_controller.rb

```diff
@@ -19,8 +19,19 @@ def create
     private
 
     def find_refresh_token
-      refresh_token_from_header = request.headers['Refresh-Token']
+      unless ApiGuard.enable_cookies_response
+        refresh_token_from_header = request.headers['Refresh-Token']
 
+        if refresh_token_from_header
```

Comment:
i think it can be also good idea to allow to set our tokens in both headers and cookies